restful.module

```diff
@@ -23,6 +22,7 @@ function restful_ctools_plugin_directory($module, $plugin) {
   if ($module == 'restful') {
     return 'plugins/' . $plugin;
   }
+  return NULL;
```

Comment:
this whole function will be removed once everything is converted to Plug, right?